### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by Maro-o** [deeztek/Hermes-Secure-Email-Gateway#49] [General]
can this SEG receive the mail and then forward it to my mailbox or anything else?
is it corrupted or i got a problem?

**Comment by deeztek** [comment]
Absolutely. Add a Relay Domain and then point it to your mailserver IP or hostname. If you set to the domain to ANY it will deliver to any recipient regardless if it exists in the Internal Recipients. If you set it to SPECIFIED then you have to have the Internal Recipient already added under Internal Recipients. You can also set Virtual Recipients that will redirect the e-mail to another mailbox inside or outside (like your gmail).

**Reply by Maro-o** [reply]
great thanks alot.
